.travis.yml

```diff
@@ -1,7 +1,7 @@
 branches:
   only:
   - master
-  - /^v[0-9.]+$/
+  - "/^v[0-9.]+$/"
```

Comment:
that regex is to inform travis to trigger build on tags which read like v0.1. tags are also sort of branches in git.